packages/grafana-data/src/themes/breakpoints.ts

```diff
@@ -0,0 +1,52 @@
+export interface ThemeBreakpointValues {
+  xs: number;
+  sm: number;
+  md: number;
+  lg: number;
+  xl: number;
+  xxl: number;
```

Comment:
That's a lot of pottential breakpoints.

Reply:
Exact same we have in current theme copied here